src/libraries/Microsoft.Extensions.Logging.Console/src/Microsoft.Extensions.Logging.Console.csproj

```
@@ -7,29 +7,24 @@
     <DefineConstants>$(DefineConstants);NO_SUPPRESS_GC_TRANSITION</DefineConstants>
     <IncludePlatformAttributes>true</IncludePlatformAttributes>
     <IsPackable>true</IsPackable>
+    <EnableConfigurationBindingGenerator>true</EnableConfigurationBindingGenerator>
+    <!-- M.E.Configuration.Binder.SourceGeneration: JavaScriptEncoder not supported; no logic generated which is okay. -->
+    <NoWarn>$(NoWarn)SYSLIB1100</NoWarn>
```

```suggestion
    <NoWarn>$(NoWarn);SYSLIB1100</NoWarn>
```

You need a separator here in case the code before you didn't put one at the end.

---

I believe this is your build error.

---

Checking, thanks.